src/node.cc

```diff
@@ -1436,7 +1436,17 @@ void FatalException(Isolate* isolate,
       exit(7);
     } else if (caught->IsFalse()) {
       ReportException(env, error, message);
-      exit(1);
+
+      // fatal_exception_function call before may have set a new exit code ->
+      // read it again, otherwise use default for uncaughtException 1
+      Local<String> exit_code = env->exit_code_string();
+      int code = process_object->Get(env->context(), exit_code)
+        .ToLocalChecked()->Int32Value(env->context()).ToChecked();
```

Comment:
nit: 4 space indent in C/C++ code

Reply:
Oops, thx. I thought linter would catch this :thinking:.

Reply:
@lundibundi Sadly, our C++ linter is very outdated and not very actively maintained … we do have https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md, if it helps?